### PR TITLE
readme_game_instructions

### PR DESCRIPTION
**Pull request by rOluochKe**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,4 +6,23 @@ This is the third project of the Main Ruby curriculum at Microverse. The goal is
...
 # 1. Usage Instructions
-Clone the repo and run the app: ruby 'bin/main.rb'. The game accepts only integers from 1 to 9. The player who succeeds in placing three of their marks in a row wins the game, or it's draw.
\ No newline at end of file
+### clone the repo: git clone https://github.com/rOluochKe/tictactoe.git
+### change directory: cd tictactoe
+### run the program: ruby 'bin/main.rb'
+### enjoy play the game
+
+# 2. Rules 
+### The game accepts only integers from 1 to 9. The player who succeeds in placing three of their marks in a row wins the game, or it's draw.
```
Comment (samgaco) [inline README.md]: Try to be more descriptive with the rules. Who starts first, how does he insert the position in which he wants to put his mark on the board, etc.

File: README.md [hunk 2]
```diff
@@ -6,4 +6,23 @@ This is the third project of the Main Ruby curriculum at Microverse. The goal is
...
+
+# 3. Winnings
```
Comment (samgaco) [inline README.md]: This should be part of the rules.

**Review by samgaco — CHANGES_REQUESTED** [review]
#### Hi @rOluochKe and @AndresFMoya :high_brightness: 

#### Let's try to make this readme file look more professional. Get some hints and inspiration in [here](https://github.com/othneildrew/Best-README-Template). 

Some personal advice:

* Focus on the styling. Now almost everything consists of bold letters and that doesn't look appealing, try to format it differently.
* When adding code, use this type of styling, makes it look way better:

```sh
$ run the program: ruby 'bin/main.rb'
```
* Imagine you are explaining on how to install your game to someone that has no idea about programming, try to baby-step a bit more and be detailed.

* Talk about the technologies that you have used to build the project
* Use this kind of styling to link to external URLs so it looks neater 

`[open issues](https://github.com/github_username/repo/issues)
`
* Check this two links if you need further inspiration [1](https://github.com/othneildrew/Best-README-Template) & [2 ](https://www.udacity.com/course/writing-readmes--ud777)

#### So please :pray: work on this and submit back for another code review when you are done.

-------------------------

Moving on have in mind :arrow_down: 

* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

-------------------------
* ###### STATUS OF THE PROJECT: REQUESTED CHANGES :stop_sign:

File: README.md (hunk 2 above)
Comment (rOluochKe) [reply README.md]: resolved

File: README.md (hunk 1 above)
Comment (rOluochKe) [reply README.md]: resolved

File: README.md [hunk 3]
```diff
@@ -1,9 +1,57 @@
...
+* [Setup](#setup)
+* [Instructions](#instructions)
+* [Winnning Patterns](#winning-patterns)
 
-# Authors
-## Raymond Oluoch https://github.com/rOluochKe/
-## Andres F. Moya https://github.com/AndresFMoya/
+## Authors
+This project was contibuted by
+* Raymond Oluoch https://github.com/rOluochKe/
```
Comment (samgaco) [inline README.md:11]: Using a format like:

Raymond Oluoch - [Github](https://github.com/rOluochKe/)

```
Raymond Oluoch - [Github](https://github.com/rOluochKe/)
```

On my opinion, would look even better, but that is a personal preference.

File: README.md [hunk 4]
```diff
@@ -1,9 +1,57 @@
...
+To run this project, install it locally using npm:
 
-# 1. Usage Instructions
-Clone the repo and run the app: ruby 'bin/main.rb'. The game accepts only integers from 1 to 9. The player who succeeds in placing three of their marks in a row wins the game, or it's draw.
\ No newline at end of file
+```
+$ clone the repo: git clone https://github.com/rOluochKe/tictactoe.git
+$ change directory: cd tictactoe
+$ run the program: ruby 'bin/main.rb'
+```
```
Comment (samgaco) [inline README.md:25]: separating the code into chunks:

```
$ clone the repo: git clone https://github.com/rOluochKe/tictactoe.git
```
```
$ change directory: cd tictactoe
```

```
$ run the program: ruby 'bin/main.rb'
```

and adding some comments in between lines, would give a better presentation and make it easier for a user with little programming knowledge.

**Review by samgaco — APPROVED** [review]
#### Hi @rOluochKe and @AndresFMoya :high_brightness: 

#### The project is approved :white_check_mark: and you may merge and submit the project completion form. 
-----------------------------------------
I would advise you to keep improving the readme file on this project or future projects moving on since it is a very important aspect of your project.

> A nice README is a good way to help people engage in the project as well. A project with nice README and screenshots will get the attention of users better since it’s a direct way to explain why this project matters, and why people should use and contribute to the project. Good README should also include enough details to help a new user get started, e.g. how to compile, how to install, and how to start integrating.

[ref](https://medium.com/@meakaakka/a-beginners-guide-to-writing-a-kickass-readme-7ac01da88ab3)


-----------------------
Moving on, remember :arrow_down: 
* **[For your next code review requests]**  Add a short summary of changes in Pull Request description

-----------------------

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**